webapp/components/Map/L.KML.js

```diff
@@ -0,0 +1,480 @@
+var L = require('leaflet')
+
+L.KML = L.FeatureGroup.extend({
```

Comment:
could you use https://github.com/windycom/leaflet-kml ?

Reply:
That is literally the same file